assets/workflow/Immunobiology/CITEseq-protocol.html

```diff
@@ -0,0 +1,1869 @@
+<html>
+
+<head>
+    <style>
+        img {
+            display: block;
+            margin-left: auto;
+            margin-right: auto;
+        }
+    </style>
+</head>
+
+<body>
+    <img src="/assets/workflow/Islet Physiology Studies/HPAP-logo.jpg"
+        alt="perifusion" width="200" height="90"
+        style="vertical-align:middle" />
+    </br>
```

Comment:
line break is ```<br>```

Reply:
@yimsheng Thanks I see the error in the output terminal of Prettier on VScode now. I guess I was using a regular Terminal before. Thanks! I will resolve these issues.